### PR TITLE
Update stream.markdown

### PR DESCRIPTION
**Pull request by tharrington1**
Made a minor grammar fix.

### CODE REVIEW

File: doc/api/stream.markdown [hunk 1]
```diff
@@ -1175,9 +1175,8 @@ as a result of this chunk.
 
 Call the callback function only when the current chunk is completely
 consumed.  Note that there may or may not be output as a result of any
-particular input chunk. If you supply as the second argument to the
-it will be passed to push method, in other words the following are
-equivalent:
+particular input chunk. If you supply a second argument it will be
```
Comment (cjihrig) [inline doc/api/stream.markdown]: Comma after `argument`

File: doc/api/stream.markdown [hunk 2]
```diff
@@ -1175,9 +1175,8 @@ as a result of this chunk.
 
 Call the callback function only when the current chunk is completely
 consumed.  Note that there may or may not be output as a result of any
-particular input chunk. If you supply as the second argument to the
-it will be passed to push method, in other words the following are
-equivalent:
+particular input chunk. If you supply a second argument it will be 
+passed to push method, in other words the following are equivalent:
```
Comment (cjihrig) [inline doc/api/stream.markdown]: `passed to the push method. In other words, the following are equivalent:`

**Comment by tharrington1** [comment]
This is just a grammar fix. Will it ever get merged?

**Comment by jasnell** [comment]
LGTM

**Comment by jasnell** [comment]
@tharrington1 ... can you squash these commits down please? I'll land once that's done.

**Comment by jasnell** [comment]
Appears to have already been fixed in nodejs/node. Closing this here. Can revisit if necessary

**Pull request closed without merging**